### PR TITLE
Final blowouts

### PR DESCRIPTION
**Pull request by devin5**
This is a pretty massive pr that involved a lot of moving parts. This pr is working and ready for QA. a-lot of these feature built in eashother. for sample. a change in main blowout to radial size, would affect the margin change I had just made for radial options which in turn changed the size I wanted the container, but all of these styling changs alone should of been there own PR. Changing either of these drastically changes the legends which was the the main goal. I really felt the only way to get this done was to do some mixing of concerns to figure out where everything could work . This page did not have a mock for it so I approach these types of things with a "try it until it looks nice attitude. I then went back and isolated what was a feature and resubmitted all of them as individual prs . 

https://github.com/4mile/rs-okr-dashboard-dev/pull/75
 https://github.com/4mile/rs-okr-dashboard-dev/pull/76 
https://github.com/4mile/rs-okr-dashboard-dev/pull/77
 https://github.com/4mile/rs-okr-dashboard-dev/pull/78
 https://github.com/4mile/rs-okr-dashboard-dev/pull/79
 https://github.com/4mile/rs-okr-dashboard-dev/pull/80
https://github.com/4mile/rs-okr-dashboard-dev/pull/81

All of these PRs are merged into this one and working...

### CODE REVIEW

File: src/components/vispage/BlowoutContainer.jsx [hunk 1]
```diff
@@ -2,24 +2,62 @@ import React, { useState, useContext, useEffect } from "react";
 import { useHistory } from "react-router-dom";
 import { makeStyles } from "@material-ui/core/styles";
 import { Grid } from "@material-ui/core";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
```
Comment (danchapman) [inline src/components/vispage/BlowoutContainer.jsx:5]: nice! I didn't know about this. Seems handy

File: src/components/vispage/BlowoutContainer.jsx [hunk 2]
```diff
@@ -47,198 +85,97 @@ function MainRadial({ time, index }) {
...
-      width: "100%",
-      
-
-    },
+  const config = {hollowSize, radialMargin, krColors, ID, pushToDashboard, setKrID, krValues}
+  const classes = krValuesExtended ? useStylesExtended() : useStylesSingle();
+  const baseOptions = krValuesExtended ? doubleOptions(config) : singleOptions(config)
+  const [visState] = useState(baseOptions);
+
+  const ExtendedView = (
```
Comment (danchapman) [inline src/components/vispage/BlowoutContainer.jsx:106]: What's the meaning of the `ExtendedView `?

Comment (danchapman) [reply src/components/vispage/BlowoutContainer.jsx:106]: like what is it visually?

**Review by danchapman — APPROVED** [review]
Overall your use of styling looks good, so if it accomplishes the look you are going for great!

File: src/index.jsx [hunk 3]
```diff
@@ -30,7 +30,7 @@ import App from "./App.jsx";
 import { Looker40SDK } from '@looker/sdk'
 window.addEventListener("DOMContentLoaded", async (event) => {
   const root = document.createElement("div");
-  root.style.overflow = "hidden";
+  // root.style.overflow = "hidden";
```
Comment (danchapman) [inline src/index.jsx:33]: remove this comment

File: src/components/vispage/options/singleOptions.js [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+          {
+            breakpoint: 1200,
+            options: {
+              legend: {
+                fontSize: 18,
+                width: 260,
+              },
+            },
+          },
+        //   {
```
Comment (danchapman) [inline src/components/vispage/options/singleOptions.js:62]: remove these comments

File: src/components/vispage/BlowoutContainer.jsx (hunk 2 above)
Comment (devin5) [reply src/components/vispage/BlowoutContainer.jsx:106]: Yeah so for me extended view is mean an extra part for a pert further than normal. here we have 4f 5 views single an done double.... extended seemed to make sense. what would you name it?

**Pull request merged**